_data/officers.yml

```diff
@@ -1,18 +1,18 @@
 # This populates the Officers page
 
 chair:
-    name: James Luther
-    contact: James.Luther
-    photo: assets/img/officers/james_luther.jpeg
-    pronouns: He/Him/His
-    bio: Jay Luther is a third-year Ph.D. student currently between advisors. He enjoys games of all kinds, reading, and hiking in the front range.
+    name: Corey Lynn Murphey
+    contact: Corey.Murphey@
+    photo: assets/img/officers/corey_murphey.jpg
+    pronouns: She/Her/Hers
+    bio: Corey is a second-year Ph.D. student interested in scientific computing, numerical computation, and computational fluid dynamics. As a classically-trained lyric soprano, Corey loves singing so much that she is trying to incorporate it into her research. In her spare time, she loves playing music, swimming, running, reading, and cuddling with her three Golden Retrievers.
```

Comment:
Can we change this to "Corey is a Ph.D. student..."

Reply:
Done @clmurphey